scripts/build_release.sh

```diff
@@ -172,7 +179,9 @@ EOF
 dpkg -l >>"${STATUSFILE}"
 gpg --clearsign "${STATUSFILE}"
 gzip "${STATUSFILE}.asc"
-aws s3 cp --quiet "${STATUSFILE}.asc.gz" "s3://algorand-devops-misc/buildlog/${RSTAMP}/${STATUSFILE}.asc.gz"
+if [ ! -z "${S3_PREFIX_BUILDLOG}" ]; then
+    aws s3 cp --quiet "${STATUSFILE}.asc.gz" "${S3_PREFIX_BUILDLOG}/${RSTAMP}/${STATUSFILE}.asc.gz"
```

Comment:
any reason this is silent but buildlog copy below (in build_release_local.sh) is not?

Reply:
this script is expected to run as a script and be recorded to a log file. `aws s3 cp` without --quiet prints out status a couple times a second and floods that log file. `build_release_local.sh` is at this time really more of a copy&paste source of things to do to run the build that get run in the build engineer's terminal.